Lib/defcon/objects/contour.py

```diff
@@ -268,8 +280,7 @@ def reverse(self):
         This will post *Contour.WindingDirectionChanged*,
         *Contour.PointsChanged* and *Contour.Changed* notifications.
         """
-        # No ufoLib analogue to ReverseContourPointPen
-        from robofab.pens.reverseContourPointPen import ReverseContourPointPen
+        from defcon.pens.reverseContourPointPen import ReverseContourPointPen
```

Comment:
can `ReverseContourPointPen` be moved to `ufoLib`? I had the same problem with `cu2qu` and ended up copying it from robofab. If it was available in `ufoLib`, I would just use that.